FreeRTOS_TCP_IP.c

```diff
@@ -423,6 +423,42 @@
     }
 /*-----------------------------------------------------------*/
 
+/**
+ * @brief Check whether the address is unicast of multicast/broadcast.
+ *
+ * @param[in] ulIPAddress: The IP address to be checked (in 32-bit format).
+ *
+ * @return pdTRUE if the IP address is either multicast or broadcast. pdFALSE
+ *         otherwise.
+ */
+    BaseType_t xIsUnicastAddress( uint32_t ulIPAddress )
```

Comment:
Although I came up with the above function signature, maybe it is better to pass a socket address:
~~~c
    BaseType_t xIsUnicastAddress( struct freertos_sockaddr * pxAddress )
    {
        uint32_t ulIPAddress = pxAddress->sin_addr;
        ...
    }
~~~
The advantage is that this will be compatible with IPv6. Would you mind to change it?